content/MBC2.md

```diff
@@ -18,13 +18,18 @@ battery to save data during power-off though. As the data consists of
 4bit values, only the lower 4 bits of the "bytes" in this memory area
 are used.
 
+### A200-BFFF - 15 "echoes" of A000-A1FF
+Only the bottom 9 bits of the address are used to index into the internal
+RAM, so RAM access repeats.
```

Comment:
As per #66, are there sources for this claim?

Reply:
Yes - the mooneye docs for one:

https://gekkio.fi/files/gb-docs/gbctr.pdf
https://github.com/Gekkio/mooneye-gb/blob/master/tests/emulator-only/mbc2/ram.s

I also found a pin diagram of the MBC2 chip - it only has address pins for the bottom 9 bits and the top 2 - the top two are used to select between ROM 0/1 areas and the cart ram, the bottom 9 index the ram. Addresses with bits 10+ set are simply invisible to the chip as it doesn't have pins for them: https://fms.komkon.org/GameBoy/Tech/Cart2.gif

Reply:
mooneye also has test roms that specifically test for this and they have been verified to pass with a real MBC2.

https://github.com/Gekkio/mooneye-gb/blob/master/tests/emulator-only/mbc2/ram.s

Reply:
Alright, though I would also copy the note that the upper 4 bits of reads are undefined *and should not be relied upon*. That's a fairly important thing to know (if for some reason one were to use MBC2).